custom_components/lennoxs30/sensor.py

```diff
@@ -139,6 +128,16 @@ def device_class(self):
     def state_class(self):
         return STATE_CLASS_MEASUREMENT
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return device info."""
+        return {
+            "name":  self._system.name,
+            "identifiers": {(DOMAIN, self._system.unique_id())},
+            "manufacturer": "Lennox",
+            "model": "Lennox S30",
+        }
+
```

Comment:
I see you are setting us up to refactor sensor into a base class by duplicating this code!  It’s a good decision to cut and paste for now as we can optimize this code in a new PR.